.github/libs/GithubUtils.js

```diff
@@ -200,13 +200,22 @@ class GithubUtils {
         deployBlockers = [],
         resolvedDeployBlockers = [],
     ) {
-        return this.octokit.pulls.list({
+        return this.octokit.paginate(this.octokit.pulls.list, {
             owner: GITHUB_OWNER,
             repo: EXPENSIFY_CASH_REPO,
             state: 'all',
+            sort: 'created',
+            direction: 'desc',
             per_page: 100,
+        }, ({data}, done) => {
+            // PRList is reverse-chronologically ordered
+            const oldestMergedPR = PRList[PRList.length - 1];
+            if (data.find(pr => pr.html_url === oldestMergedPR)) {
+                done();
+            }
+            return data;
         })
-            .then(({data}) => {
+            .then((data) => {
```

Comment:
Are you sure we don't need this destructuring anymore?

Reply:
yes we don't need it